data/guidelines/en/corrections_and_others.md

```diff
@@ -5,4 +5,4 @@ of segmentation into zones and lines.
 
 For in-line corrections, we recommend using the combination of "⟦" [U+27E6] and "⟧" [U+27E7] to encapsulate a legible word marked as crossed out. When the word is not decipherable (because of a stain, an incomplete tracing or any other reason), "⟦" and "⟧" can be used without any letter in between ("⟦⟧"). <!-- AC: I almost want to say that this is also this symbol that should be used to mark transcription uncertainty during the transcription -->
 
-For corrections relying on an interlinear addition, we first recommend that the interlinear addition be considered a separate text line, type accordingly using the SegmOnto vocabulary. In cases where a symbol is used in the source to mark where the interlinear addition should be inserted, we recommend to always use the same unicode character : "⎀" [U+2380] for modern and contemporary sources. <!-- AC: I think it's a different symbol for medieval docs: "‸" [U+2038] -->
+For corrections relying on an interlinear addition, we first recommend that the interlinear addition be considered a separate text line, type accordingly using the SegmOnto vocabulary. In cases where a symbol is used in the source to mark where the interlinear addition should be inserted, we recommend to always use the same unicode character : "⎀" [U+2380] for modern and contemporary sources but "‸" [U+2038] for medieval sources.
```

Comment:
Is is really necessary to have two different approaches here ?

Reply:
No you're right, we could use the same symbol as catmus medieval.

We had agreed, with Simon and Ariane, that `"⎀" [U+2380]` is technically more correct than `"‸" [U+2038]` since it is an "insertion symbol". 

Now, it's easy to fix for catmus modern since it's not released yet.

Reply:
ok, pour simplifier, je bascule CATMuS Modern v1 vers `‸`